packages/odyssey-react/src/components/Select/Select.module.scss

```diff
@@ -221,15 +282,15 @@
     padding-inline: 0;
     transform: translateY(-50%);
     opacity: 1;
-    background-color: cv("blue", "400");
+    background-color: var(--ColorBackgroundButton);
     background-image: url("data:image/svg+xml;charset=utf-8,%3Csvg viewBox='0 0 14 14' fill='none' xmlns='http://www.w3.org/2000/svg' class='ods-icon'%3E%3Cpath d='M2.323 10.118a1.102 1.102 0 1 0 1.559 1.56L7 8.558l3.118 3.118a1.102 1.102 0 0 0 1.56-1.559L8.558 7l3.118-3.118a1.102 1.102 0 0 0-1.559-1.56L7 5.442 3.882 2.323a1.102 1.102 0 1 0-1.56 1.559L5.442 7l-3.118 3.118Z' fill='%23fff'/%3E%3C/svg%3E");
     background-position: center;
     background-size: ($base-line-height * 1em) - 1em;
```

Comment:
Can we turn this into a `calc()` with `FontLineHeightBase`? I know we're going to address our `calc()` usage in the future, but it'll get some Sass out of our code.

Reply:
I didn't want to open this up to theming as it is part of what allows the background SVG to function correctly here yes?

Reply:
The line-height here should match the line-height applied to the text surrounding the icon, so we'd want them to share the same token. That way, if someone updates their body line-height, the background-size will increase along with the text.

Reply:
@edburyenegren-okta got it new PR up now